contracts/axelar-gateway/src/auth.rs

```diff
@@ -43,11 +43,7 @@ pub fn validate_proof(env: &Env, data_hash: BytesN<32>, proof: Proof) -> bool {
 
     let signers_hash: BytesN<32> = env.crypto().keccak256(&signers_set.to_xdr(env)).into();
 
-    let signers_epoch: u64 = env
-        .storage()
-        .persistent()
-        .get(&DataKey::EpochBySignerHash(signers_hash.clone()))
-        .unwrap_or(0);
+    let signers_epoch: u64 = get_signers_epoch(env, &signers_hash);
 
     if signers_epoch == 0 {
```

Comment:
remove the use of `0` and check for `None` directly, you can unwrap via a let

`let Some(epoch) = env...get(DataKey::Epoch) else { panic_with_error }`

Reply:
got it, updated code.